mindsdb/integrations/handlers/statsforecast_handler/requirements.txt

```diff
@@ -0,0 +1 @@
+statsforecast>=1.4.0
```

Comment:
Maybe cap it to `<2.0` so that it doesn't suddenly break once the Nixtla guys get there.

Reply:
Great shout, will change.